en/mkdocs.yml

```diff
@@ -251,7 +251,11 @@ nav:
                 - HTTP 2.0: deploy-and-publish/deploy-on-gateway/choreo-connect/http-2.0.md
                 - Rate Limiting:
                     - Enabling distributed throttling: deploy-and-publish/deploy-on-gateway/choreo-connect/rate-limiting/distributed-throttling.md
-                - Service Discovery: deploy-and-publish/deploy-on-gateway/choreo-connect/service-discovery.md
+                - Service Discovery:
+                    - Configure Choreo Connect with Consul: deploy-and-publish/deploy-on-gateway/choreo-connect/service-discovery/service-discovery.md
+                    - Service Discovery:  deploy-and-publish/deploy-on-gateway/choreo-connect/service-discovery/choreo-connect-and-service-discovery.md
```

Comment:
Shall we make this the first topic? We can rename this to Service Discovery Overview.

Reply:
I changed **Configure Choreo Connect with Consul** --> **Service Discovery Overview**
Guess that's what you meant
5378648

Reply:
I meant change Service Discovery (service-discovery.md) to Service Discovery Overview.